config/desktop/trixie/environments/mate/armbian/create_desktop_package.sh

```
@@ -0,0 +1,164 @@
+# install lightdm greeter
+cp -R "${SRC}"/packages/blobs/desktop/lightdm "${destination}"/etc/armbian
```

Shellcheck (0.10.0)

[warning] destination is referenced but not assigned.

(SC2154)